src/components/CodeAssignmentModal/EmailTemplate.jsx

```diff
@@ -0,0 +1,12 @@
+const EMAIL_TEMPLATE = `Your learning manager has provided you with a new access code to take a course at edX. You may redeem this code for {{REDEMPTIONS_REMAINING}} # of courses.
```

Comment:
We'll still need to figure out the correct interpretation of the token in this line, but we can keep as is for now!